cocotb/share/lib/gpi_log/gpi_logging.c

```diff
@@ -145,54 +155,65 @@ void gpi_log(const char *name, long level, const char *pathname, const char *fun
     if (level < local_level)
         return;
 
-    // Ignore truncation
-    // calling args is level, filename, lineno, msg, function
-    //
     PyGILState_STATE gstate = PyGILState_Ensure();
 
-    PyObject *check_args = PyTuple_New(1);
-    PyTuple_SetItem(check_args, 0, PyLong_FromLong(level));
+    PyObject *filename_arg = NULL, *lineno_arg = NULL, *msg_arg = NULL, *function_arg = NULL;
```

Comment:
Probably worth a comment like `declared here in order to be initialized before any goto statements`

Reply:
Added comment.